lib/breadcrumbs_on_rails/railtie.rb

```diff
@@ -10,7 +10,10 @@ module BreadcrumbsOnRails
 
   class Railtie < Rails::Railtie
     ActiveSupport.on_load(:action_controller) do
-      include BreadcrumbsOnRails::ActionController
+      # There is no need to include breadcrumbs on an API controller
+      unless self.name == 'ActionController::API'
```

Comment:
No, there is no inheritance: https://github.com/rails/rails/blob/b326e82dc012d81e9698cb1f402502af1788c1e9/actionpack/lib/action_controller/api.rb#L88